src/pages/ProfilesExplorerView/components/SceneLabelValuesTimeseries.tsx

```diff
@@ -71,9 +73,55 @@ export class SceneLabelValuesTimeseries extends SceneObjectBase<SceneLabelValues
             })
         )
         .setHeaderActions(headerActions(item))
+        .setMenu(
+          new VizPanelMenu({
+            items: [
+              {
+                text: 'Scale',
+                type: 'group',
+                subMenu: [
+                  {
+                    text: 'Logarithmic (2)',
```

Comment:
Does the "(2)" here refer to log base 2? If so, is that relevant? Maybe we can drop it. I spent a bit trying to figure out what "(2)" meant. If there's no other log options to select I think it makes sense to not include it.

Reply:
Base 10 is also a common option, I think it's important to be specific. 
I'd go for `Logarithmic (base 2)` (vernose)  or `Log2`.